dome9/resource_dome9_ruleset_test.go

```diff
@@ -99,7 +99,6 @@ resource "%s" "%s" {
 	cloud_vendor       = "aws"
 	language           = "en"
 	hide_in_compliance = false
-	is_template        = false
```

Comment:
I think you are not align with master, please align

Reply:
No, that ok